extensions/adobe/experience/adcloud/searchads/adgroup.schema.json

```diff
@@ -5,39 +5,39 @@
     "you may not use this file except in compliance with the License. You may obtain a copy",
     "of the License at https://creativecommons.org/licenses/by/4.0/"
   ],
-  "$id": "https://ns.adobe.com/experience/adcloud/searchadvertising/adgroup",
+  "$id": "https://ns.adobe.com/xdm/adcloud/searchads/adgroup",
   "$schema": "http://json-schema.org/draft-06/schema#",
   "title": "Search Advertising AdGroup",
   "type": "object",
-  "meta:extensible": true,
-  "meta:abstract": true,
+  "meta:extensible": false,
+  "meta:abstract": false,
   "meta:auditable": true,
   "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
   "description": "Search Advertising Ad Group is a logical collection or grouping of ads.",
   "definitions": {
-    "searchadvertising-adgroup": {
+    "searchads-adgroup": {
```

Comment:
@jishah-adobe can you please justify the reasoning for adding adgroup name and id under searchad namespace Vs adcloud name space so that these becomes reusable between searchad and dsp

Reply:
DSP and Search have semantically different meanings thought entity/schema names may be alike.  In this specific case Ad Group is relevant to Search Advertising only hence searchads. Entity Ad Group would not be defined for dsp